README.md

```diff
@@ -215,6 +223,204 @@ objectives:
+- **condition**, required field. Defines the conditions of the alert
+  - **kind** *enum(burnrate, guard, custom)* the kind of alerting condition thats checked, defaults to `burnrate`
+
+If the kind is `burnrate` the following fields are required:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **lookbackWindow** *number*, required field, the time-frame for which to calculate the threshold
+- **controlLookbackWindow** *number*, optional field
+- **alertAfter** *number*: required field, the duration the condition needs to be valid, defaults `0m`
+
+If the kind is `custom` the following fields are required:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **comparison** *enum(lt, lte, gt, gte, eq)*, optional field, defines
+  how the threshold should be compared to meet the threshold, defaults to `gt`
+
+If the kind is `guard` the following fields are required, can  be used to
+define quality conditions:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **criteriaType** *enum(pass, warning)*, required field, defines the criteria type of the condition
+- **weight** *number*, required field, the weight or importance of the condition
+
+---
+
+An example of a alert condition is the following:
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertCondition
+metadata:
+  name: cpu-usage-breach
+  displayName: CPU usage breach
+spec:
+  description: If the CPU usage is too high for given period then it should alert
+  severity: page
+  condition:
+    kind: burnrate
+    threshold: 0.9
+    lookbackWindow: 1h
+    alertAfter: 5m
+```
+
+---
+
+#### Alert Notification Target
+
+An Alert Notification Target defines the possible targets where alert notifications
+should be delivered to. For example, this can be a web-hook, Slack or similar
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertNotificationTarget
+metadata:
+  name:
+  displayName: string # optional, human readable name
+spec:
+  target: # required
+  description: # optional
+  parameters: # optional, list of parameters for the notification target
+    - name: string # required
+      description: string # optional
+      type: string # required, string, secret, number, url
+      value: string # optional
+      defaultValue: string # optional
+```
+
+An example of the Alert Notification Target can be:
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertNotificationTarget
+metadata:
+  name: OnCallDevopsMailNotification
+spec:
+  description: Notifies by a mail message to the on-call devops mailing group
+  target: email
+  parameters:
+    - name: emailAddress
+      type: email
+      value: on-call-devops@openslo.org
+```
+
+Alternatively, a similar notification target can be defined for Slack in this example
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertNotificationTarget
+metadata:
+  name: OnCallDevopsSlackNotification
+spec:
```

Comment:
I imagined that you could use these notification targets to generate some configuration like for Alertmanage, for example:

```
receivers:
  - name: oncalldevopsslacknotification
    slack_configs:
      - api_url: https://hooks.slack.com/services/XXX1XXXXX/XXXXXXX8X/acdifghf7Klxy8xZCEDXyOk5I
        channel: '#alerts'
        send_resolved: true
        text: "A priority one alert has been triggered for the service $service_name for SLO $slo_name,
         please extinguish the fire, thank you!
         Don't forget to schedule a meeting with the company board via Executive Secretary"
```